extension.json

```
@@ -1,9 +1,9 @@
 {
     "name": "Rosé Pine",
-    "version": "1.0.4",
+    "version": "1.0.5",
     "authors": [
         "Kainoa Kanter <kainoa@t1c.dev>"
     ],
-    "description": "soho vibes for zed",
+    "description": "Theme soho vibes for zed",
```

This should match our brand a little more:

```suggestion
    "description": "All natural pine, faux fur, and a bit of soho vibes – a classy minimalist theme for Zed.",
```

Haven't had to worry about this before because most apps differentiate themes from other extensions so I'm open to suggestions. @ThatOneCalculator has the final say on this one 😉